src/RobotSimulator/StdWorldBuilder.ts

```diff
@@ -75,6 +75,20 @@ export class StdWorldBuilder {
 
     const robotSpec = robotBuilder.generateSpec();
     robotSpec.initialPosition = this.startPosition;
+    robotSpec.customMesh = {
+      filePath: `${process.env.PUBLIC_URL}/assets/models/robot_full_compressed.glb`,
+      rotation: {
+        x: 0,
+        y: 0,
+        z: 0,
+      },
+      scale: {
+        x: 0.2,
```

Comment:
did you check the sizes of the model matches this? the debug mode should help you decide this

Reply:
Nope, will do now